test/onnx/test_operators.py

```diff
@@ -892,6 +893,50 @@ def test_softmaxcrossentropy_4d(self):
         y = torch.empty(3, 2, 1, dtype=torch.long).random_(5)
         self.assertONNX(torch.nn.CrossEntropyLoss(), (x, y), opset_version=12)
 
+    def test_lstm_none_sequence_lens(self):
+        input = torch.randn(RNN_SEQUENCE_LENGTH, BATCH_SIZE, RNN_INPUT_SIZE)
```

Comment:
I don't understand what the test name means. In particular, I don't see anything set to `None`. Could you please add a doc string or rename the test to be clearer?

Reply:
doc string updated